doc/manual/syntax.md

```diff
@@ -302,33 +304,33 @@ The following examples show how symbolic strings are desugared:
 
 #### Enum tags
 
-Enumeration tags are used to express finite alternatives. They are formed by
-writing a backtick `` ` `` followed by any valid identifier. For example,
-`std.serialize` takes an export format as a first argument, which is an enum
-tag among `` `Json ``, `` `Toml `` or `` `Yaml `` (as of version 0.1):
+Enumeration tags are used to express a choice among finitely many alternatives.
+They are formed by writing a backtick `` ` `` followed by any valid identifier
+or by a quoted string. For example, `std.serialize` takes an export format as a
+first argument, which is an enum tag among `` `Json ``, `` `Toml `` or `` `Yaml
+``:
 
 ```nickel
-std.serialize `Json {foo = 1}
-# gives "{
-#          \"foo\": 1
-#        }"
-std.serialize `Toml {foo = 1}
-# gives "foo = 1
-#       "
+> std.serialize `Json {foo = 1}
+"{
+   \"foo\": 1
+ }"
+
+> std.serialize `Toml {foo = 1}
+"foo = 1
+"
 ```
 
 An enum tag `` `foo `` is serialized as the string `"foo"`:
 
 ```nickel
-let as_yaml_string = std.serialize `Yaml {foo = `bar}
-# gives "---
-#       foo: bar
-#       "
-
+> std.serialize `Yaml {foo = `bar}
```

Comment:
Nitpick: maybe JSON would actually do a better job at showing that it is a string